.github/workflows/dryrun.yml

```diff
@@ -35,4 +35,30 @@ jobs:
     - name: Install kas
       run: sudo pip3 install kas
     - name: DryRun sdv-image-all
-      run: kas build kas/leda-kirkstone.yaml -- --dry-run
+      run: kas build kas/leda-qemux86-64.yaml -- --dry-run
+
+  dryrun-qemuarm64:
+    name: Dry-Run (qemuarm64)
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install tools
+      run: sudo apt-get install -y --no-install-recommends socat file gawk wget git diffstat unzip texinfo gcc build-essential chrpath socat cpio python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev pylint xterm python3-subunit mesa-common-dev zstd liblz4-tool tmux mc skopeo fdisk ruby-full jq libvirt-clients libvirt-daemon-system qemu-system-x86 qemu-system-arm qemu-kvm squashfs-tools rauc python3-newt ca-certificates curl gnupg lsb-release
+    - name: Install kas
+      run: sudo pip3 install kas
+    - name: DryRun sdv-image-all
+      run: kas build kas/leda-qemuarm64.yaml -- --dry-run
+
+  dryrun-raspberrypi4-64:
+    name: Dry-Run (raspberrypi4-64)
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install tools
+      run: sudo apt-get install -y --no-install-recommends socat file gawk wget git diffstat unzip texinfo gcc build-essential chrpath socat cpio python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev pylint xterm python3-subunit mesa-common-dev zstd liblz4-tool tmux mc skopeo fdisk ruby-full jq libvirt-clients libvirt-daemon-system qemu-system-x86 qemu-system-arm qemu-kvm squashfs-tools rauc python3-newt ca-certificates curl gnupg lsb-release
```

Comment:
Some of qemu-* stuff in not needed i think

Reply:
Good finding, i'll remove the packages which are not needed